x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_creation_ui/components/step_define_rule/index.tsx

```diff
@@ -393,14 +393,16 @@ const StepDefineRuleComponent: FC<StepDefineRuleProps> = ({
                 id="xpack.securitySolution.dataViewSelectorText1"
                 defaultMessage="Use Kibana "
               />
-              <DocLink guidePath="kibana" docPath="data-views.html" linkText="Data Views" />
+              <DocLink
+                docPath="explore-analyze/find-and-organize/data-views"
+                linkText="Data Views"
+              />
               <FormattedMessage
                 id="xpack.securitySolution.dataViewSelectorText2"
                 defaultMessage=" or specify individual "
               />
               <DocLink
-                guidePath="kibana"
-                docPath="index-patterns-api-create.html"
+                docPath="api/doc/kibana/group/endpoint-data-views"
```

Comment:
I just want to confirm: is this the approach we're taking in general for the `Index Patterns`/`Data Views` migration? 

This messaging specifies "data views OR index patterns," and previously linked to each, respectively. Updating "index patterns" to point to the data views page seems like it could cause confusion, especially since it makes no mention of the deprecations (as [the old index patterns pages did](https://www.elastic.co/guide/en/kibana/7.17/index-patterns-api.html)).

Reply:
Yes, agreed that this is a step in the right direction 👍